scipy/optimize/_qap.py

```diff
@@ -50,21 +50,21 @@ def quadratic_assignment(A, B, method="faq", options=None):
     options : dict, optional
         A dictionary of solver options. All solvers support the following:
 
-        partial_match : 2d-array of integers, optional, (default = None)
+        maximize : bool (default: False)
```

Comment:
I agree there should not be a comma before the parentheses. Do you know if a standard for specifying defaults like this (comma vs colon) is written anywhere? It's not in the [Guide to NumPy/SciPy Documentation](https://numpy.org/doc/stable/docs/howto_document.html). This is fine; I'm just wondering if you know of a standard.

Reply:
I don't know of a standard. I grep the codebase and look for the most common pattern. In this case there is no pre-existing `optional, (...`.
On the other hand, there is only a single instance of `optional (...`

Reply:
Yeah, there's no question about the comma; it shouldn't have one. Oops, I mean to ask:
> Do you know if a standard for specifying defaults like this (**equals sign** vs colon) is written anywhere?

Reply:
I don't know of a standard for that either.

Grep shows that there are 11 modules which use colon. Only `scipy/optimize/_linprog_ip.py` uses an equals sign.

Reply:
Oops, my bad in `_linprog_ip.py`. I guess colon is the de facto standard.